kaiax/builder/impl/tx_pool_wrapping.go

```diff
@@ -87,6 +87,21 @@ func (b *BuilderWrappingModule) IsReady(txs map[uint64]*types.Transaction, next
 	}
 	if isReady && b.txBundlingModule.IsBundleTx(tx) {
 		if _, ok := b.knownTxs[tx.Hash()]; !ok {
+			if maxBundleSize := b.txBundlingModule.GetMaxBundleSize(); maxBundleSize > 0 {
+				flattened := make([]*types.Transaction, 0, len(b.knownTxs))
+				for _, tx := range b.knownTxs {
+					if !tx.tx.IsMarkedUnexecutable() {
+						flattened = append(flattened, tx.tx)
+					}
+				}
+				// it's too much cost to check the size of the bundle here, so we just check the number of txs
+				// if the number of txs is greater than the max bundle size, we reject the tx
+				// but if there are ready txs, we should add the tx to the pool to complete the bundle
+				if len(flattened) >= maxBundleSize && len(ready) == 0 {
```

Comment:
```suggestion
				numExecutable := 0
				for _, tx := range b.knownTxs {
					if !tx.tx.IsMarkedUnexecutable() {
						numExecutable++
					}
				}
				// it's too much cost to check the size of the bundle here, so we just check the number of txs
				// if the number of txs is greater than the max bundle size, we reject the tx
				// but if there are ready txs, we should add the tx to the pool to complete the bundle
				if numExecutable > maxBundleSize && len(ready) == 0 {
```

Reply:
fix at https://github.com/kaiachain/kaia/pull/328/commits/f83b2d735d0c7e78a5dcf706bc12b71053543a61

Reply:
`numExecutable > maxBundleSize` is not fixed

Reply:
Why numExecutable > maxBundleSize?

Reply:
Usually max size (or cap or limit) is inclusive, right?

Reply:
I updated the limitation logic throughout.